src/components/Footer.tsx

```diff
@@ -46,35 +46,32 @@ export const Footer = ({ navbar, logo }) => {
                 <h2 className="mb-3 font-bold">Regulations</h2>
                 <ul>
                   <li className="mb-2">
-                    <a href="#" className="hover:underline">
-                      Privacy Policy
-                    </a>
+                    {/* <a href="#" className="hover:underline"> */}
+                    Privacy Policy
+                    {/* </a> */}
                   </li>
                   <li className="mb-2">
-                    <a href="#" className="hover:underline">
-                      Terms &amp; Conditions
-                    </a>
+                    {/* <a href="#" className="hover:underline"> */}
+                    Terms &amp; Conditions
+                    {/* </a> */}
                   </li>
                   <li>
-                    <a href="#" className="hover:underline">
-                      Cookie settings
-                    </a>
+                    {/* <a href="#" className="hover:underline"> */}
+                    Cookie settings
+                    {/* </a> */}
                   </li>
                 </ul>
               </div>
               <div>
                 <h2 className="mb-3 font-bold">Contacts</h2>
                 <ul>
                   <li className="mb-2">
-                    <a
-                      href="mailto:someone@example.com"
-                      className="hover:underline "
-                    >
+                    <a href={`mailto:${contacts[0]?.email}`} className="hover:underline ">
                       Email
                     </a>
                   </li>
                   <li>
-                    <a href="" className="hover:underline">
+                    <a href={contacts[0]?.linkedIn} className="hover:underline">
```

Comment:
I think it should work with `contacts.linkedIn`

Reply:
ah, ok, I checked, the schema is different 

![Uploading localhost_3000.png…]()